frontend/src/components/commons/Notepad.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from "react";
+import LOCALIZE from "../../text_resources";
+import TextareaAutosize from "react-textarea-autosize";
+import { BROWSER_STRING, IE_STRING, VALID_BROWSERS } from "../../Status";
+
+let COLS = "";
+
+const styles = {
+  windowPadding: {
+    paddingTop: 43,
+    paddingLeft: 20
+  },
+  h4: {
+    color: "white"
+  },
+  title: {
+    paddingLeft: 15,
+    paddingTop: 0.1,
+    height: 55,
+    backgroundColor: "#00565e",
+    borderRadius: "5px 5px 0 0"
+  },
+  content: {
+    backgroundColor: "white",
+    border: "1px solid #00565e",
+    borderRadius: "5px 5px 0 0",
+    width: 220,
+    height: "calc(100vh - 208px)"
+  },
+  notepadSection: {
+    overflow: "auto",
+    height: "calc(100vh - 270px)"
+  },
+  textArea: {
+    resize: "none",
+    border: "none"
+  },
+  center: {
+    textAlign: "center"
+  }
+};
+
+class Notepad extends Component {
+  //Adjust the notepad text zone width depending on the browser
+  detectBrowser = () => {
+    //if browser is IE
+    if (BROWSER_STRING === IE_STRING) {
+      COLS = "20";
```

Comment:
Do these need to be strings, or can they be numbers?

Reply:
We can also use numbers.
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/113/commits/a96d0f64235ca0daae98f4665f53349f33f34f4e